.codeclimate.yml

```diff
@@ -25,7 +26,7 @@ checks:
     enabled: true
 plugins:
   rubocop:
-    enabled: true
-    channel: rubocop-0-80
+    enabled: false
+    channel: rubocop-0-67
```

Comment:
Why are we reverting back to 0-67 and disabling rubocop?

Reply:
Undid this change